packages/stencil/src/components/datacom-display-card/datacom-display-card-group.css

```diff
@@ -0,0 +1,25 @@
+@import url("../../global/css/breakpoints.css");
+
+:host {
+  display: inline;
+}
+
+.dc-display-card-group {
+  column-gap: 48px;
+  display: grid;
+  row-gap: 32px;
+}
+
+@media (--breakpoint-tablet) {
+  .dc-display-card-group {
+    grid-template-columns: 1fr;
```

Comment:
I'm assuming tablet should just have the cards one on top of each other? If so, that would be happening by default the way you've written, so you don't need to define the `grid-template-columns` OR the `grid-template-rows`